### PR TITLE
feat: add support for zkSync networks

### PR DESCRIPTION
**Pull request by therealemjy**
## JIRA tickets

- VEN-2711

## Changes

- add support for zkSync Sepolia and zkSync mainnet network

### CODE REVIEW

File: apps/evm/package.json [hunk 1]
```diff
@@ -114,13 +114,13 @@
...
-    "@venusprotocol/oracle": "^2.3.0",
+    "@venusprotocol/governance-contracts": "^2.3.0-dev.4",
+    "@venusprotocol/isolated-pools": "^3.5.0-dev.5",
+    "@venusprotocol/oracle": "^2.5.0-dev.3",
     "@venusprotocol/stylelint-config": "*",
-    "@venusprotocol/token-bridge": "^2.1.0",
+    "@venusprotocol/token-bridge": "^2.2.0-dev.1",
     "@venusprotocol/typescript-config": "*",
-    "@venusprotocol/venus-protocol": "^9.1.0",
+    "@venusprotocol/venus-protocol": "^9.2.0-dev.8",
```
Comment (gleiser-oliveira) [inline apps/evm/package.json]: Do we want to merge with the dev versions?

Comment (therealemjy) [reply apps/evm/package.json]: Not yet indeed, I'll keep this as a draft

**Review by gleiser-oliveira — APPROVED** [review]

File: apps/evm/src/libs/tokens/infos/commonTokens/zkSyncMainnet.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    asset: usdtLogo,
+  },
+  {
+    address: '0x3355df6d4c9c3035724fd0e3914de96a5a83aaf4',
+    decimals: 6,
+    symbol: 'USDC.e',
+    asset: usdcLogo,
+  },
+  {
+    address: '0x8A2E9048F5d658E88D6eD89DdD1F3B5cA0250B9F',
```
Comment (chechu) [inline apps/evm/src/libs/tokens/infos/commonTokens/zkSyncMainnet.ts]: This is the ZK asset on sepolia. ZK is `0x5a7d6b2f92c77fad6ccabd7ee0624e64907eaf3e` on mainnet

Comment (chechu) [reply apps/evm/src/libs/tokens/infos/commonTokens/zkSyncMainnet.ts]:
```suggestion
    address: '0x5a7d6b2f92c77fad6ccabd7ee0624e64907eaf3e',
```

**Pull request merged**